questions/119-medium-replaceall/template.ts

```diff
@@ -2,4 +2,8 @@ export type ReplaceAll<
   S extends string,
   From extends string,
   To extends string
-> = any
+> = From extends ''
+  ? S
+  : S extends `${infer F}${From}${infer L}`
+  ? `${F}${To}${ReplaceAll<L, From, To>}`
```

Comment:
Works great.

What would happen if I do:

```suggestion
  ? `${ReplaceAll<L, From, To>}${F}${To}`
```

I know that's not gonna meet the requirement but, why?